### PR TITLE
Bug Fix: navigation bug in side drawer menu

### LINKED ISSUE #2529: ROUTES.TABBAR.PROFILE not opening from userAvatarView
ROUTES.TABBAR.PROFILE not opening from userAvatarView

Screenshot 2022-11-07 at 4 35 19 PM

### PR DESCRIPTION
**Pull request by aliseyalvi**
### What does this PR?
This PR fixes the issue of navigation when user avatar is pressed from side drawer menu. 

### Issue number
fixes #2529 

### Screenshots/Video

https://user-images.githubusercontent.com/48380998/201531075-0acd6fba-8e7a-4764-b111-883ff3789c02.mp4

### CODE REVIEW

File: src/components/collapsibleCard/view/collapsibleCardView.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React, { PureComponent } from 'react';
 import { View, TouchableHighlight } from 'react-native';
-import Animated, { EasingNode } from 'react-native-reanimated';
-
+import { View as AnimatedView } from 'react-native-animatable';
```
Comment (noumantahir) [inline src/components/collapsibleCard/view/collapsibleCardView.js:3]: @aliseyalvi please revert this...

**Review by noumantahir — CHANGES_REQUESTED** [review]

File: src/components/collapsibleCard/view/collapsibleCardView.js (hunk 1 above)
Comment (aliseyalvi) [reply src/components/collapsibleCard/view/collapsibleCardView.js]: @noumantahir changes done https://github.com/ecency/ecency-mobile/pull/2530/commits/09a930489b5ae19f7d1b7eb84b5800e8cc67b80f

**Review by noumantahir — APPROVED** [review]
lgtm

**Review by feruzm — APPROVED** [review]
Lgtm

**Pull request merged**